src/main/java/carracing/domain/Winners.java

```diff
@@ -0,0 +1,28 @@
+package carracing.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+
+    private final List<Car> winners;
+    private int maximumDistance;
+
+    public Winners(List<Car> cars) {
+        maximumDistanced(cars);
+
+        this.winners = cars.stream()
+                .filter(car -> car.getPosition() == maximumDistance)
+                .collect(Collectors.toList());
+    }
+
+    public List<Car> getWinners() {
+        return this.winners;
+    }
+
+    public void maximumDistanced(List<Car> cars) {
+        for (Car car : cars) {
+            this.maximumDistance = Math.max(this.maximumDistance, car.getPosition());
+        }
+    }
```

Comment:
```suggestion
    private final List<Car> winners;

    public Winners(List<Car> cars) {
        getMaximumDistance(cars);

        this.winners = cars;
    }

    public List<Car> getWinners() {
        int maximumDistance = getMaximumDistance(this.winners);

        return this.winners.stream()
                .filter(car -> car.getPosition() == maximumDistance)
                .collect(Collectors.toList());
    }

    private int getMaximumDistance(List<Car> cars) {
        return cars.stream()
                .mapToInt(Car::getPosition)
                .max()
                .orElseThrow(NoSuchElementException::new);
    }
```
이렇게 만들어보면 어떨까요?
아래의 두가지 관점을 기준으로 수정해보았어요. :)

- 멤버 변수의 사용을 줄인다.
- 우승자들의 목록을 구하는 시점은 getWinners()이 호출 될 때 해보면 어떨까?

Reply:
Winner 객체 초기화시 반드시 우승자만 있어야 한다는 고정관념에서 벗어날 수 있는 코드였습니다!
멤버변수도 줄어서 깔끔하고 좋은 것 같습니다.
덧붙여 람다를 잘 사용하고 싶네요..

Reply:
지석님이라면 경험이 조금 쌓이신다면 충분히 잘 사용하실 수 있을꺼에요!!